src/__tests__/rate.test.js

```diff
@@ -176,4 +176,33 @@ describe('rate', () => {
       { mu: 25, sigma: 8.204837030780652 },
     ])
   })
+
+  it('accepts a tau term', () => {
+    expect.assertions(1)
+    const a = rating({ mu: 25, sigma: 3 })
+    const b = rating({ mu: 25, sigma: 3 })
+    const [[winner], [loser]] = rate([[a], [b]], {
+      tau: 0.3,
+    })
+
+    expect([winner, loser]).toStrictEqual([
+      { mu: 25.624880438870754, sigma: 2.9879993738476953 },
+      { mu: 24.375119561129246, sigma: 2.9879993738476953 },
+    ])
+  })
+
+  it('prevents sigma from rising', () => {
+    expect.assertions(1)
+    const a = rating({ mu: 40, sigma: 3 })
+    const b = rating({ mu: -20, sigma: 3 })
+    const [[winner], [loser]] = rate([[a], [b]], {
+      tau: 0.3,
+      preventSigmaIncrease: true,
+    })
```

Comment:
I think 90-10 is probably not enough. In a very high skill game a much better player's win percentage is very nearly 100%

Reply:
I just ran a simulation where players can have pretty dramatic skill differences to the point where there are 100% win rates. Amazingly out of 100k matches, rating went down on a win a total of 11806 times.

Here are the relevant lines in the script that count these scenarios (I'm sorry for how messy this script is):
https://gist.github.com/JLaferri/ccde20693ff2953ea7cf578ce60ac8d3#file-openskilltesting-js-L135-L141

Also worth noting that I'm fairly confident that in practice in my system and game, these scenarios absolutely will happen, the ratings that I'm seeing now in production seem to indicate as such.

One reason maybe to not have the option on would be if you intend to change your tau. If you increase your tau, I think with time sigmas would re-stabilize at a higher point. If you increase your tau but don't allow increases, however, then sigma will stay locked in at the low steady state it was at with the old tau.

Reply:
Cool, thanks for this! I think my simulation never encountered this because I only had 2 players, which was naive of me.

I'll merge this but I'd love to come up with something more numerically continuous; I have a hunch there should be some sort of solution involving a `gamma` function that considers `Z`, and would prevent this situation from within the model itself.